pkg/uptimerobot_test.go

```diff
@@ -75,66 +75,120 @@ func TestUnmarshalMonitor(t *testing.T) {
 
 func TestCreate(t *testing.T) {
 	t.Parallel()
-	client := New("dummy")
-	// force test coverage of the client's dump functionality
-	client.Debug = ioutil.Discard
-	ts := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if r.Method != http.MethodPost {
-			t.Errorf("want POST request, got %q", r.Method)
-		}
-		wantURL := "/v2/newMonitor"
-		if r.URL.EscapedPath() != wantURL {
-			t.Errorf("want %q, got %q", wantURL, r.URL.EscapedPath())
-		}
-		body, err := ioutil.ReadAll(r.Body)
-		if err != nil {
-			t.Fatal(err)
-		}
-		r.Body.Close()
-		want, err := ioutil.ReadFile("testdata/requestNewMonitor.json")
-		if err != nil {
-			t.Fatal(err)
-		}
-		// Convert the received body and the expected body to maps, for
-		// ease of comparison
-		wantMap := map[string]interface{}{}
-		err = json.Unmarshal(want, &wantMap)
-		if err != nil {
-			t.Fatal(err)
-		}
-		bodyMap := map[string]interface{}{}
-		err = json.Unmarshal(body, &bodyMap)
-		if err != nil {
-			t.Fatal(err)
-		}
-		if !cmp.Equal(wantMap, bodyMap) {
-			t.Error(cmp.Diff(wantMap, bodyMap))
-		}
-		w.WriteHeader(http.StatusOK)
-		data, err := os.Open("testdata/newMonitor.json")
-		if err != nil {
-			t.Fatal(err)
-		}
-		defer data.Close()
-		io.Copy(w, data)
-	}))
-	defer ts.Close()
-	client.HTTPClient = ts.Client()
-	client.URL = ts.URL
-	create := Monitor{
-		FriendlyName:  "My test monitor",
-		URL:           "http://example.com",
-		Type:          TypeHTTP,
-		Port:          80,
-		AlertContacts: []string{"3", "5", "7"},
-	}
-	got, err := client.CreateMonitor(create)
-	if err != nil {
-		t.Error(err)
+
```

Comment:
Just a house style tip; I don't use blank lines within functions in this project.